theforeman.org/pipelines/release/source/foreman.groovy

```diff
@@ -8,98 +8,61 @@ pipeline {
         buildDiscarder(logRotator(daysToKeepStr: '7'))
     }
 
+    environment {
+        BUNDLE_WITHOUT = 'development'
+        RAILS_ENV = railsEnvForTask(TASK)
+        DATABASE_URL = databaseUrlForTask(TASK, env)
```

Comment:
I suspect you're not allowed to pass in `env` here:
```
java.lang.IllegalArgumentException: One or more variables have some issues with their values: DATABASE_URL
```

Thoughts on just using some UUID for the DB name?